docs/_sass/_main.scss

```diff
@@ -182,6 +183,190 @@ button {
     align-content: center;
 }
 
+.logo {
+    width: 80%;
+}
+
+.search-icon {
+    margin: auto 0px;
+}
+
+.flex {
+    display: -webkit-box;
+    display: -moz-box;
+    display: -ms-flexbox;
+    display: -moz-flex;
+    display: -webkit-flex;
+    display: flex;
+    -webkit-flex-flow: row wrap;
+    flex-flow: row wrap;
+    align-content: space-between;
+}
+
+#sidebar-search {
+    background-color: $color-green-appBG;
+    width: 400px;
+    height: 100vh;
+    position: fixed;
+    display: block;
+    top: 0;
+    right: 0;
+}
+
+@media only screen and (max-width: $breakpoint-tablet) {
+    #sidebar-search {
+        width: 100%;
+    }
+}
+
+.searchbar-title-wrapper {
+    padding: 20px;
+}
+
+.search-title {
+    font-size: 120%;
```

Comment:
Why are we using 120% for the title? We use 17px in NewDot for these headers, let's standardize on that.

Reply:
Noted, done it